dashboard/app/bisect_test.go

```diff
@@ -225,6 +234,34 @@ https://goo.gl/tpsmEJ#testing-patches`,
 			bisectLogLink, bisectCrashReportLink, bisectCrashLogLink))
 	}
 
+	// BisectFix #3
+	pollResp = c.client2.pollJobs(build.Manager)
+	c.expectNE(pollResp.ID, "")
+	c.expectEQ(pollResp.Type, dashapi.JobBisectFix)
+	c.expectEQ(pollResp.ReproOpts, []byte("repro opts 3"))
+	c.advanceTime(5 * 24 * time.Hour)
```

Comment:
We should start getting fix bisections only after 30 days, but this test only advances time by 5 days. Does this test pass? Why?

Reply:
Fixed, this was because of a typo in the query.